src/System.Threading.ThreadPool/tests/ThreadPoolTests.cs

```diff
@@ -30,10 +30,13 @@ public static void ConcurrentInitializeTest()
                 {
                     countdownEvent.Signal();
                     countdownEvent.Wait(ThreadTestHelpers.UnexpectedTimeoutMilliseconds);
-                    Assert.True(ThreadPool.SetMinThreads(processorCount, processorCount));
                 };
 
+            // ThreadPool.SetMinThreads returns false on UapAot
```

Comment:
UapAot uses the default Win32 thread pool, which does not allow setting min/max numbers.  I suggested using a custom Win32 thread pool to allow setting min/max numbers, however, that idea was not approved.  We need evidence that these settings may be important for real-world applications.